tests/conftest.py

```diff
@@ -114,3 +114,52 @@ def json(self):
             return {'token': 'superlegittoken'}
 
     mocker.patch("requests.post", return_value=FakeExternalResponse())
+
+@pytest.fixture(scope='function')
+def fake_invalid_auth_from_oc(mocker):
+    """
+    Changes the return value of requests.post to be a custom response
+    object so that we aren't validating external APIs in our unit tests
+    """
+    class ExternalApiRequest(object):
+        @classmethod
+        def post(cls):
+            return FakeExternalResponse()
+
+    class FakeExternalResponse(object):
+        @classmethod
+        def json(self):
+            # Mock an error returned from the OC backend
+            return {'error': 'Invalid Email or password.'}
+
+    mocker.patch("requests.post", return_value=FakeExternalResponse())
+
+@pytest.fixture(scope='function')
+def fake_items_error(mocker):
+    """
+    Intentionally raises an error to test error response is correct
```

Comment:
These descriptions should be more like the above one. Why do we mock this method, what are we mocking. 

If the mock is specific to a single usage, I'm unsure if intended usage should be included.